private/ft_inv.m

```diff
@@ -45,127 +64,175 @@
 %
 %    FieldTrip is distributed in the hope that it will be useful,
 %    but WITHOUT ANY WARRANTY; without even the implied warranty of
-%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%    MERCHANTABILITY or FITNESS FOR X PARTICULAR PURPOSE.  See the
```

Comment:
did you do a search replace all for the input argument? X -> A

Reply:
well caught. Yes, my bad.